capabilities.json

```diff
@@ -116,6 +116,14 @@
                         }
                     }
                 },
+                "fontFamily": {
+                    "displayName": "Font Family",
```

Comment:
You need to add `displayNameKey` property. Then add the string with correspond name key in [string resources](https://github.com/microsoft/powerbi-visuals-sankey/blob/master/stringResources/en-US/resources.resjson). It used by localization service in Power BI.  

For example heatmap visual [has localization for the same property](https://github.com/microsoft/powerbi-visuals-heatmap/blob/master/stringResources/en-US/resources.resjson#L16)

And It also has correspond strings for other languages. For example:
German: https://github.com/microsoft/powerbi-visuals-heatmap/blob/master/stringResources/de-DE/resources.resjson#L13
Russian: https://github.com/microsoft/powerbi-visuals-heatmap/blob/master/stringResources/ru-RU/resources.resjson#L13
e.t.c.

**It will be nice if you copy them from heatmap visual and add to the sankey string resources.**

You can [read about localization in documentation](https://docs.microsoft.com/en-us/power-bi/developer/visuals/localization#how-to-localize-capabilities)

Reply:
Gotcha. Just added all the localization strings to the appropriate files.